data/officers.json

```diff
@@ -6,8 +6,8 @@
     },
     "vicepresident": {
         "position": "Vice President",
-        "name": "Nicole Anderson-Au",
-        "image": "nicole.jpg"
+        "name": "Stephen Parkinson",
+        "image": "stephen.jpg"

```

Comment:
These should probably be calpoly usernames, it will prevent name collisions in the future